src/reference/asciidoc/testing.adoc

```diff
@@ -100,6 +100,110 @@ A convenient constant `KafkaEmbedded.SPRING_EMBEDDED_KAFKA_BROKERS` is provided
 With the `KafkaEmbedded.brokerProperties(Map<String, String>)` you can provide additional properties for the Kafka server(s).
 See https://kafka.apache.org/documentation/#brokerconfigs[Kafka Config] for more information about possible broker properties.
 
+Starting with _version 1.3.1_, the embedded broker is also compatible with kafka version 1.0.0.
+Use the following in your pom to override the versions in a Spring Boot application:
+
+[source, xml]
+----
+<dependency>
+	<groupId>org.springframework.kafka</groupId>
+	<artifactId>spring-kafka</artifactId>
+	<version>1.3.1.RELEASE</version>
```

Comment:
I think there is `{spring-kafka-version}` instead of hard-coded version what makes it sticky and incompatible during cherry-pick.

Reply:
Cool; forgot about that.